packages/gestalt/src/SheetMobile.js

```diff
@@ -0,0 +1,183 @@
+   */
+  onAnimationEnd?: ({| animationState: 'in' | 'out' |}) => void,
+  /**
+   * Callback fired when SheetMobile is dismissed. Must be used for controlling SheetMobile's visibility state.
+   */
+  onDismiss: () => void,
+  /**
+   * The main SheetMobile content section has a "default" padding. For those cases where full bleed is needed, set `padding` to "none".
+   */
+  padding?: 'default' | 'none',
+  /**
+   * Adds an primary action Button for user interaction at the end of the header section. See the [header variant, with primary action case](https://gestalt.pinterest.systems/web/sheetmobile#Header) for more info.
+   */
+  primaryAction?: {|
+    accessibilityLabel: string,
+    href?: string,
+    label: string,
+    onClick: OnClickType,
+    rel?: $ElementType<ElementConfig<typeof Link>, 'rel'>,
+    size?: $ElementType<ElementConfig<typeof Button>, 'size'>,
+    target?: $ElementType<ElementConfig<typeof Link>, 'target'>,
+  |},
+  /**
+   * The underlying ARIA role for the SheetMobile. See the [Accessibility Role section](https://gestalt.pinterest.systems/web/sheetmobile#Role) for more info.
+   */
+  role?: 'alertdialog' | 'dialog',
+  /**
+   * Shows a dismiss button on Popover. See the [header variant, dismiss button case](https://gestalt.pinterest.systems/web/sheetmobile#Header) for more info.
+   */
+  showDismissButton?: boolean,
+  /**
+   * Subheading for SheetMobile. See the [header variant](https://gestalt.pinterest.systems/web/sheetmobile#Header) for more info.
+   */
+  subHeading?: string,
+  /**
+   * Sets the SheetMobile's height. See the [size variant](https://gestalt.pinterest.systems/web/sheetmobile#Size) for more info.
+   */
+  size?: 'default' | 'full' | 'auto',
+|};
+
+/**
+ * [SheetMobile](https://gestalt.pinterest.systems/web/sheetmobile) is a mobile only component. It is not used in desktop experiences.
+ *
+ * SheetMobile is a supplementary container that sits on top of the screen’s primary content and can be dismissed in order to interact with the underlying content. Sheets can contain a wide variety of information and layouts, including menu items, actions, and supplemental content.
+ *
+ *
+ * ![SheetMobile light mode](https://raw.githubusercontent.com/pinterest/gestalt/master/playwright/visual-test/SheetMobile.spec.mjs-snapshots/SheetMobile-chromium-darwin.png)
+ * ![SheetMobile dark mode](https://raw.githubusercontent.com/pinterest/gestalt/master/playwright/visual-test/SheetMobile-dark.spec.mjs-snapshots/SheetMobile-dark-chromium-darwin.png)
+ *
+ */
+function SheetMobile({
+  accessibilityLabel,
+  align = 'start',
+  backIconButton,
+  children,
+  closeOnOutsideClick = true,
+  forwardIconButton,
+  onAnimationEnd,
+  onDismiss,
+  footer,
+  padding,
+  primaryAction,
+  heading,
+  role,
+  showDismissButton = true,
+  subHeading,
+  size = 'default',
+}: Props): Node {
+  const deviceType = useDeviceType();
+  const isMobile = deviceType === 'mobile';
+
+  if (!isMobile) return null;
```

Comment:
oh I see, we can't use it on non-mobile. Should this be logged? Or Console.error("SheetMobile: Trying to use sheet-mobile on a non-mobile device")

Reply:
DONE